e2e/tests-dfx/rust.bash

```diff
@@ -20,10 +20,30 @@ teardown() {
     assert_command dfx build hello
     assert_match "ic-cdk-optimizer not installed"
     cargo install ic-cdk-optimizer
+    # shellcheck disable=SC2030
     export PATH="$HOME/.cargo/bin/:$PATH"
     assert_command dfx build hello
     assert_match "Executing: ic-cdk-optimizer"
     assert_command dfx canister --no-wallet install hello
     assert_command dfx canister --no-wallet call hello greet dfinity
     assert_match '("Hello, dfinity!")'
 }
+
+@test "rust canister can resolve dependencies" {
+    dfx_new_rust rust_deps
+    install_asset rust_deps
+
+    # TODO: cargo install may takes too much time
+    cargo install ic-cdk-optimizer
```

Comment:
I think this is failing on CI because both tests try to install ic-cdk-optimizer.